examples/chart/teleport-cluster/README.md

```diff
@@ -43,3 +46,8 @@ Check https://goteleport.com/teleport/docs for more details.
 | `acmeURI`                 | ACME server to use for certificates                                         | `https://acme-v02.api.letsencrypt.org/directory`       | no       |
 | `labels.[name]`           | Key-value pairs, for example `--labels.env=local --labels.region=us-west-1` |                                                        | no       |
 | `enterprise`              | Use Teleport Enterprise                                                     | `false`                                                | no       |
+
+## Guides
+
+See https://goteleport.com/docs/kubernetes-access/helm/guides/ for guides on setting up HA Teleport clusters
```

Comment:
Nit: do you mind using `[]()` notation for linking to another article, `https://URL...` for an example endpoint (like the Web UI or `...`/webapi/ping`)?